kuma/core/tests/test_middleware.py

```diff
@@ -1,11 +1,14 @@
-from mock import MagicMock
+from mock import patch, MagicMock
 
 from django.test import RequestFactory
 
 from kuma.core.tests import KumaTestCase, eq_
 from kuma.core.middleware import (
+    WhiteNoiseMiddleware,
```

Comment:
As we are using `RestrictedWhiteNoiseMiddleware` which inherit `WhiteNoiseMiddleware`, is it necessary to use the `WhiteNoiseMiddleware`?

Reply:
If `WhiteNoiseMiddleware` should be used, it should be import from `whitenoise.middleware`

Reply:
@safwanrahman I'm importing and using `WhiteNoiseMiddleware` only because I'm patching it's `process_request` method with a `MagicMock` instance.

Reply:
@escattone I think in that case it should be imported from `whitenoise.middleware`

Reply:
I'd normally write it the way @safwanrahman suggested, but because of the usage (check if the parent class is called), this way works too, and may be preferable. No change.